index.html

```diff
@@ -504,7 +504,11 @@ <h2>
         at least its <a>origin</a>, including whether it is served over a
         secure connection. This UI SHOULD differ from any UI used when the
         [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        that the user is navigating off scope. In addition, the user agent
+        SHOULD reset the <a>application context</a>'s <a>default theme
+        color</a> to its default value. The off-scope document may override
```

Comment:
Is its default value defined? or are you referring to "a User Agent defined default value"?

Reply:
Updated. Thanks!